src/components/elements/Loading.tsx

```diff
@@ -11,7 +11,7 @@ const Loading = () => (
     aria-live='polite'
     aria-busy='true'
   >
-    <CircularProgress size={50} />
+    <CircularProgress aria-label='Loading' size={50} />
```

Comment:
Is there a reason we don't just use the usual `<Loading/>` here?

Reply:
This _is_ the `Loading` component 🙂

Reply:
Ack sorry hahaha you are right, I was rushing, but meant to put this comment [here](https://github.com/greenriver/hmis-frontend/pull/935/files#diff-12793900bb3fac98fe746720c86e1a33888ef8fd9b188976a0ed7c9026ab6e1bR54) below on the `ClientEnrollmentCard`.

It looks like there are a few places in our app where we use MUI's `CircularProgress` outside of our own `Loading` component; looks like often these are in the 'disabled' color as opposed to the usual primary color.

Reply:
I see, I assume that was just before we added the Loading component. My IDE is showing that ClientEnrollmentCard CircularProgress was  last updated 2 years ago, so that's probably it.

Reply:
Ok sounds good. I will add a note to maybe consolidate these when we update the code to match the Design Library.